addon/validations/result.js

```diff
@@ -68,6 +69,21 @@ var ValidationsObject = Ember.Object.extend({
       }
     }
     return !isNone(attrValue);
+  }),
+
+  error: computed('message', 'isInvalid', 'attribute', function() {
+    if (get(this, 'isInvalid')) {
+      return ValidationError.create({
+        message: get(this, 'message'),
+        attribute: get(this, 'attribute')
+      });
+    } else {
+      return null;
+    }
+  }),
+
+  errors: computed('error', function() {
```

Comment:
This isnt needed here. A result will only have a single error unless the _validations object get replaced by a validation-result object (only in belongs-to and has-many). Thats why the messages CP exists here but no actually messages prop.